build.sbt

```diff
@@ -614,8 +614,25 @@ lazy val bench = project.in(file("test") / "benchmarks")
   ).settings(inConfig(JmhPlugin.JmhKeys.Jmh)(scalabuild.JitWatchFilePlugin.jitwatchSettings))
 
 
+// at present (March 2019) there is very little code in here, because it's new.
+// in addition to providing a place for shared code, this also lets the JUnit
+// dependency flow into the partest subproject, allowing partests to (for example)
+// use the JUnit assert APIs (an also as yet little-used capability, because also new)
```

Comment:
we could probably live without that comment :-)

Reply:
removed